Library/Homebrew/cask/cmd/audit.rb

```diff
@@ -43,7 +44,7 @@ def run
         end
         casks = casks.map { |c| CaskLoader.load(c, config: Config.from_args(args)) }
         any_named_args = casks.any?
-        casks = Cask.to_a if casks.empty?
+        casks = Cask.all if casks.empty?
```

Comment:
previous discussion: https://github.com/Homebrew/brew/pull/14827#discussion_r1119405825

Reply:
@dduugg Can you check this isn't throwing a deprecation warning when it's not stubbed by `allow`?

Reply:
Oh, it does, I just assumed that was intentional:
```
  1) Cask::Cmd::Audit selection of Casks to audit audits all Casks if no tokens are given
     Failure/Error: described_class.run

     MethodDeprecatedError:
       Calling Cask::Cask#all without --all or HOMEBREW_EVAL_ALL is deprecated! There is no replacement.
     # ./cask/cask.rb:31:in `all'
     # ./cask/cmd/audit.rb:47:in `run'
     # ./cask/cmd/abstract_command.rb:62:in `run'
     # ./test/cask/cmd/audit_spec.rb:17:in `block (3 levels) in <top (required)>'
     # ./test/support/helper/spec/shared_context/homebrew_cask.rb:52:in `block (2 levels) in <top (required)>'
```

Note that the existing code would throw `NoMethodError` absent the `allow`:
```
  1) Cask::Cmd::Audit selection of Casks to audit audits all Casks if no tokens are given
     Failure/Error: described_class.run

     NoMethodError:
       undefined method `to_a' for Cask::Cask:Class
       Did you mean?  to_s
     # ./cask/cmd/audit.rb:47:in `run'
     # ./cask/cmd/abstract_command.rb:62:in `run'
     # ./test/cask/cmd/audit_spec.rb:17:in `block (3 levels) in <top (required)>'
     # ./test/support/helper/spec/shared_context/homebrew_cask.rb:52:in `block (2 levels) in <top (required)>'
```

What would you like to do here?

Reply:
@dduugg This file has now been removed in https://github.com/Homebrew/brew/pull/15201 so that should solve the problem after a `git rebase origin/master`.